my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-info.html

```diff
@@ -133,7 +133,7 @@ <h2 class="md-headline">Emergency contact</h2>
             <!-- repeat for each emergency contact -->
             <emergency ng-repeat="contact in emergencyInfo"></emergency>
           </md-list>
-          <div layout="row">
+          <div ng-hide="emergencyInfo.length > 2" layout="row">
```

Comment:
Clever. I'd be interested in e.g. @mrdahman input about whether this yields a sufficiently clear experience or if users will not understand where the "Add another" UI control went.

Reply:
@apetro , @Doug-Reed did ask for my opinion; I believe the pattern is clear, that the user has reached the capacity of contacts. Of course, I'm willing to hear feedback from the help desk that students *want* four emergency contacts; but I really feel that this was a bug pointed out by bored help desk agents and doesn't really have serious user impact.